etc/conf/wget-template

```diff
@@ -413,6 +413,16 @@ md5sum_() {
     fi
 }
 
+#Our own sha256sum function call that takes into account machines that don't have sha256sum but do have sha2 (i.e. mac os x)
+sha256sum_() {
+    hash -r
+    if type sha256sum >& /dev/null; then
+        echo $(sha256sum $@)
+    else
+        echo $(sha2 -q -256 $@)
```

Comment:
This command doesn't exist in my MacOSX. 
I would suggest to replace with:
`        echo $(openssl sha256 "$@" | awk '{print $2}') ` 

see comment 2 on issue #84